src/Raven.Server/Commercial/LicenseManager.cs

```diff
@@ -154,6 +154,15 @@ public void Initialize(StorageEnvironment environment, TransactionContextPool co
             }
         }
 
+        public void PutMyNodeInfoAfterClusterVersionSetting(object sender, AfterClusterVersionSettingEventArgs args)
+        {
+            if (args.PreviousClusterVersion >= args.CurrentClusterVersion)
+                return;
+
+            Task.Run(PutMyNodeInfoAsync).IgnoreUnobservedExceptions();
+            ClusterCommandsVersionManager.AfterClusterVersionSetting -= PutMyNodeInfoAfterClusterVersionSetting;
```

Comment:
I do not think this is necessary?

Reply:
Considering the context provided earlier, I propose that we should promptly unsubscribe from this event after executing `PutMyNodeInfoAsync`. This is because our intention was to trigger this event just a single time during the initialization phase.